astropy/cosmology/utils.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from math import inf
+
+import numpy as np
+
+from astropy.utils import isiterable
+
+__all__ = []  # nothing is publicly scoped
+
+
+def _float_or_none(x, digits=3):
```

Comment:
Perhaps without underscore, since it is used outside of the module, but again, that's super nitpicky.

Reply:
I was considering this, but this PR is just about moving functions. I'd prefer not to semi-scope this function. At least not yet. I would like to eventually deprecate this and the ``vectorize_if_needed`` func.

Reply:
I was considering this, but this PR is just about moving functions. I'd prefer not to semi-scope this function. At least not yet. I would like to eventually deprecate this func and ``vectorize_if_needed``.